queries.sql

```diff
@@ -1,3 +1,43 @@
 /*Queries that provide answers to the questions from all projects.*/
+SELECT *
+FROM animals
+WHERE name LIKE '%mon%';
```

Comment:
- [ ]  `LIKE '%mon%'` means Finds any values that have "mon" in any position, but According to the requirements we need to Find all animals whose name ends in "mon"., So it would be better to use `LIKE '%mon'` 💯

![image](https://user-images.githubusercontent.com/45231276/196305791-fdf272d4-3663-4b27-975d-38ecda93a1a5.png)

Reply:
totally missed that😅😅 @Bondok6  Thanks for the review man! I'll make sure to do the required changes

Reply:
well resolved! ✔️